### PR TITLE
[FIX] l10n_fr: add tax move lines in tax report

### PR DESCRIPTION
**Pull request by gteboul**
[FIX] l10n_fr: add tax move lines in tax report
In the l10n_fr tax report, the wrong `account_move_line` entries are used in the journal item list view of the tax report in section `TD-VAT Due`. As a result, the balance debit/credit displayed under the `TD-VAT Due` is not the same as the balance in the journal items list view.

- Create a customer invoice with a price 1000 and a tax 20% Goods.
- Create a vendor bill with a price 100 and a tax 20% Goods.
- Open the tax report, under the TD - VAT Due, the balance is 180.
- Click on the amount in TD - VAT Due, the debit is 20 but the credit is 1000 (should be 200).

The formula for `TD-VAT Due` is defined as `box_16.balance - box_23.balance`.
- `box_16` is the sum of box_xx.balance which are defined by `account_tag`.
- The balances of these boxes were computed using the base `account_tag`, which calculates the correct amount but causes `_expand_aggregations` in `account_report_expression` to generate a list view based on base move lines instead of the corresponding tax lines. This can be seen the label field of the lines, which are not the tax label.

`box_16` needs to be computed using `box_xx_taxe` instead of `box_xx_base` to ensure the correct tax lines are included.
For box 08, 09, 9B, 10, 11, T1, T2, T3, T4, T5, T6, P1, P2, I1, I2, I3, I4, I5, I6 we replace balance to balance_from_tags (which uses box_xx_taxe)
For box 13, 14, 15, T7, 5B we keep balance (which already uses box_xx_taxe)

opw-4335396

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_fr/data/tax_report_data.xml [hunk 1]
```diff
@@ -962,7 +962,7 @@
...
-                                <field name="aggregation_formula">box_08_taxe.balance + box_09_taxe.balance + box_9B_taxe.balance + box_10_taxe.balance + box_11_taxe.balance + box_13_taxe.balance + box_14_taxe.balance + box_T1_taxe.balance + box_T2_taxe.balance + box_T3_taxe.balance + box_T4_taxe.balance + box_T5_taxe.balance + box_T6_taxe.balance + box_T7_taxe.balance + box_P1_taxe.balance + box_P2_taxe.balance + box_I1_taxe.balance + box_I2_taxe.balance + box_I3_taxe.balance + box_I4_taxe.balance + box_I5_taxe.balance + box_I6_taxe.balance + box_15.balance + box_5B.balance</field>
+                                <field name="aggregation_formula">box_08_taxe.balance_from_tags + box_09_taxe.balance + box_9B_taxe.balance + box_10_taxe.balance + box_11_taxe.balance + box_13_taxe.balance + box_14_taxe.balance + box_T1_taxe.balance + box_T2_taxe.balance + box_T3_taxe.balance + box_T4_taxe.balance + box_T5_taxe.balance + box_T6_taxe.balance + box_T7_taxe.balance + box_P1_taxe.balance + box_P2_taxe.balance + box_I1_taxe.balance + box_I2_taxe.balance + box_I3_taxe.balance + box_I4_taxe.balance + box_I5_taxe.balance + box_I6_taxe.balance + box_15.balance + box_5B.balance</field>
```
Comment (yosa-odoo) [inline addons/l10n_fr/data/tax_report_data.xml:965]: isn't it an issue for the other taxes ?

In the ticket, they talk about a 20% taxes (invoice and bill)

But what about an invoice 20% and a bill 5.5%

**Review by yolannsabaux — COMMENTED** [review]
I don't really know french taxes but see comment

File: addons/l10n_fr/data/tax_report_data.xml (hunk 1 above)
Comment (gteboul) [reply addons/l10n_fr/data/tax_report_data.xml:965]: For the tax lines related to the bills (box_23), it seems that there is no issue. However, for some reason, invoices with 20% goods (services) do not appear at all in the tax report, but I am not sure if this is intended.

**Comment by yosa-odoo** [comment]
Hello Accounting, I let you see what's going on in here. Juvr has left so don't know who to ping

**Comment by lordkrandel** [comment]
France, so... 
immagine

**Comment by proose** [comment]
@gteboul Hello, I've got an error testing your runbot (box_T7_taxe.balance_from_tags doesn't exist, and others may not exist either)

**Review by proose — APPROVED** [review]
Thank you for your work ! Another reviewer will come a final look and ultimately r+ :)

**Comment by FlorianGilbert** [comment]
@Olivier-olma May I ask you your opinion on this PR
Thank you

**Review by FlorianGilbert — APPROVED** [review]
Discussed with @Olivier-olma 
LGTU

@robodoo r+

**Pull request closed without merging**